registration/templates/navbar.html

```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="{% url 'home'%}">OK Merseburg</a>
```

Comment:
The name of the company should be configurable from the settings.

Reply:
[2a757b](https://github.com/gocept/ok_tools/pull/16/commits/2a757b60a3fc7934eb3a03c1aef92523503e03a8)